application/dash_application/views/bcl2fastq.py

```diff
@@ -170,17 +169,16 @@ def update_layout(run_alias):
             run_alias, index
         )
 
-        pie_data, textarea_fraction = create_pie_chart(
-            run, pruned, total_clusters
-        )
+        pie_data, textarea_fraction = create_pie_chart(run, pruned, total_clusters)
 
         return (
             create_known_index_bar(run),
             create_unknown_index_bar(pruned),
             pie_data,
-            textarea_fraction
+            textarea_fraction,
```

Comment:
extra comma?

Reply:
Yes, to minimize the diff. If more items are added to the structure, it will be a one-line change to add the new item as opposed to two lines, one with added comma and one with the new item.